ksiren-okhttp3-plugin/src/main/java/com/brightspace/ksiren/okhttp3_plugin/KSirenOkhttp3RequestBuilder.kt

```diff
@@ -50,37 +48,15 @@ class KSirenOkhttp3RequestBuilder(wrappedAction: Action): KSirenRequestBuilder<R
 	private fun getBody(): RequestBody {
 		when (action.type) {
 			ContentType.JSON -> {
-				val mediaType = MediaType.parse(action.type.value)
-
-				val json = StringBuilder()
-				json.append("{")
-
-				var count = 0
-				fieldValues.forEach { (name, value) ->
-					if (count > 0) {
-						json.append(", ")
-					}
-
-					json.append("\"")
-					json.append(name)
-					json.append("\": \"")
-					json.append(value)
-					json.append("\"")
-
-					count += 1
-				}
-
-				json.append("}")
-				return RequestBody.create(mediaType, json.toString())
+				return RequestBody.create(MediaType.parse(action.type.value), action.toJsonRequestBody())
```

Comment:
I'm seeing a nullability mismatch here re: `action.toJsonRequestBody()` can return null, but `RequestBody.create()` doesn't accept null (via annotation because it's Java...if it were Kotlin it wouldn't compile at all).

Reply:
Worst-case, I'll just return empty braces as would have been provided before in the "zero field" case.